OpenRA.Mods.Common/Traits/Attack/AttackBase.cs

```diff
@@ -161,18 +161,10 @@ protected virtual void OnStopOrder(Actor self)
 
 		static Target TargetFromOrder(Actor self, Order order)
```

Comment:
It seems this entire method is completely unused?

Reply:
It's used from 5 places...?